loop/storage/redis.js

```diff
@@ -940,6 +957,7 @@ RedisStorage.prototype = {
       'userRooms.' + userMac,
       'room.' + roomToken
     );
+    multi.expire('userRooms.' + userMac, self._settings.hawkSessionDuration);
```

Comment:
actually, I'm not sure we should expire these. What happens if someone tries to join again after 1 month, and he doesn't find its rooms anymore?